lib/tsc-watch.js

```diff
@@ -11,7 +9,7 @@ const stringArgv = require('string-argv');
 
 const compilationStartedRegex = /Starting incremental compilation/;
 const compilationCompleteRegex = /( Compilation complete\. Watching for file changes\.| Found \d+ error[s]?\. Watching for file changes\.)/;
-const typescriptSuccessRegex = /Compilation complete/;
+const typescriptSuccessRegex = /Compilation complete|Found 0 errors/;
```

Comment:
"Found 0 errors"?

Reply:
Ah yes, In my case it never showed a green message, had to add that to the regexp so it showed this message green at least:

![image](https://user-images.githubusercontent.com/2250701/51490955-a8bb0800-1d8b-11e9-944c-acd95bce5c76.png)

Do you think is not worth it?

Reply:
As you might noticed we already have something similar `compilationCompleteRegex`.
Maybe we can use that?

Reply:
Well... I thought of that but what if the compilation result is more than zero errors? then it should not be green and that regex allows for that. I believe it is for a different purpose.

Reply:
it's a Regex with \d+ in it, so it'll catch all numbers. anyways the error catching is inside an `if` before.

Reply:
What I mean is that the `compilationCompleteRegex` regex is not used in the code to color the messages. It is used for something else, and anyway, because of that \d+ it does not work for the coloring, because we only want it green if there are zero errors, if there are more than zero I don't know, maybe show it red? but there is another regex that shows TS errors in red already.

Reply:
ok, let's leave it as it is , I'll have to go over it and rethink this.
We'll use your addition.